include/erlcloud_aws.hrl

```diff
@@ -6,9 +6,11 @@
           s3_scheme="https://"::string(),
           s3_host="s3.amazonaws.com"::string(),
           s3_port=80::non_neg_integer(),
+          s3_follow_redirect=false::boolean(),
```

Comment:
this if for other feature namely support for S3 follow redirect. please amend it for this PR